packages/components/src/components/ActionsDropdown/ActionsDropdown.js

```diff
@@ -39,7 +44,47 @@ class RunDropdown extends Component {
 
   render() {
     const { modal = {}, showDialog } = this.state;
-    const { intl, items, resource } = this.props;
+    const { intl, isButton, items, resource } = this.props;
+
+    const dialog = showDialog ? (
+      <Modal
+        open={showDialog}
+        modalHeading={modal.heading}
+        primaryButtonText={modal.primaryButtonText}
+        secondaryButtonText={
+          modal.secondaryButtonText ||
+          intl.formatMessage({
+            id: 'dashboard.modal.cancelButton',
+            defaultMessage: 'Cancel'
+          })
+        }
+        onRequestClose={this.handleClose}
+        onRequestSubmit={this.handleModalAction}
+        onSecondarySubmit={this.handleClose}
+        danger={modal.danger}
+      >
+        {modal.body && modal.body(resource)}
+      </Modal>
+    ) : null;
+
+    if (isButton && items.length === 1) {
```

Comment:
This is so that we can still use this component on the list views for other resources which only support a single action (delete) but where we still want the overflowmenu.

I'm not a huge fan of the `isButton` naming though, so welcome any suggestions.

Reply:
`useButton`? `useBtn`? `renderAsButton`? `renderAsBtn`? Kind of conflicts with the component name of `ActionsDropdown` though

Reply:
naming is hard… 😅  Maybe the component should be called `Actions` (no mention of how it's rendered) and the prop could be `kind="button"` (defaults to "overflowmenu")? Although `kind` in Carbon is usually reserved for things like button variants (`primary`, `ghost`, etc.). Maybe `mode="inline"` or similar?

Reply:
`Actions` + `kind` seems good. The kind is specific to the component, so I dont think it conflicts with carbon's scheme. Carbon uses different set of kind values for Notifications for example

Reply:
Done